src/polymer/js-utils.ts

```diff
@@ -146,30 +146,26 @@ export function toScannedMethod(
 export function getOrInferPrivacy(
     name: string,
     annotation: jsdoc.Annotation|undefined,
-    privateUnlessDocumented: boolean): Privacy {
+    inferFromName = true,
```

Comment:
Explicit privacy annotations always win so they could add an `@private` comment if they don't want to name them with underscores, but IMO underscores are typically communicating privacy.